src/metaschema/oscal_implementation-common_metaschema.xml

```diff
@@ -665,7 +665,7 @@
             <!-- This is an id because the idenfier is assigned and managed by humans. -->
             <formal-name>System Identification</formal-name>
             <!-- Identifier Declaration -->
-            <description>A <a href="/concepts/identifier-use/#human-oriented">human-oriented</a>, <a href="/concepts/identifier-use/#globally-unique">globally unique</a> identifier with <a href="/concepts/identifier-use/#cross-instance">cross-instance</a> scope that can be used to reference this system identification property elsewhere in <a href="/concepts/identifier-use/#scope">this or other OSCAL instances</a>. When referencing an externally defined <code>system identification</code>, the <code>system identification</code> must be used in the context of the external / imported OSCAL instance (e.g., uri-reference). This string should be assigned <a href="/concepts/identifier-use/#consistency">per-subject</a>, which means it should be consistently used to identify the same system across revisions of the document.</description>
+            <description>A <a href="/concepts/identifier-use/#globally-unique">globally unique</a> identifier with <a href="/concepts/identifier-use/#cross-instance">cross-instance</a> scope that can be used to reference this system identification property elsewhere in <a href="/concepts/identifier-use/#scope">this or other OSCAL instances</a>. When referencing an externally defined <code>system identification</code>, the <code>system identification</code> must be used in the context of the external / imported OSCAL instance (e.g., uri-reference). This string should be assigned <a href="/concepts/identifier-use/#consistency">per-subject</a>, which means it should be consistently used to identify the same system across revisions of the document.</description>
```

Comment:
I am unsure of my recommendation here, but there can be one of two outcomes in my opinion. Feedback welcome. As it stands, in the `allowed-values/enum`, we have system identifer types that are both human oriented _or_ machine-oriented like UUIDs (RFC-4122)

1. We suggest this is not _only_ a human-oriented identifier in the documentation string, because we recommend both a type for human-oriented and machine-oriented.
1. We update the enumeration values
1. Do nothing and leave it is as-is.

1 is easier, and 2 might be more disruptive and less than ideal.

Key Model Review Criteria:

This criteria is given approach 1. 2 is more complex, 3 obviously none of it applies. :-)

- Is this proposed change to 1) the model itself, 2) documentation embedded in the model, or 3) both? 2
- Is this proposed change to 1) enhance the model in short-term way that will not break backwards compatibility, 2) a significant change that will require breaking backwards compatibility or 3) it is unclear, and I need to provide explanation? 2
- Is this proposed change to 1) current operational use of one model or 2) cross-document mechanics or 3) not change the operational use of the models? 3

Reply:
We had some good constructive discussion about this, and the documentation approach is ill-advised. What would be preferred, as part of the docs with machine readable attributes in the Metaschema source. We had talked about that, but we need to further that in Metaschema (per https://github.com/usnistgov/metaschema/issues/222). Also moving forward, the equivalent change that will result here is not approach 1, leave it out.

I will roll back the change and leave a comment, but we need to further https://github.com/usnistgov/metaschema/issues/222 before we can have a different approach that is explicit and not implicit (like approach 1).